### PR TITLE
chore: update baileys-version.json

### PR DESCRIPTION
**Pull request by RzkGnz**

### CODE REVIEW

File: src/Defaults/baileys-version.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-	"version": [2, 2308, 7]
-}
\ No newline at end of file
+	"version": [2, 2321, 6]
```
Comment (edgardmessias) [inline src/Defaults/baileys-version.json]:
```suggestion
	"version": [2, 2322, 15]
```

Comment (edgardmessias) [reply src/Defaults/baileys-version.json]: Could you change to latest?

**Review by edgardmessias — CHANGES_REQUESTED** [review]

File: src/Defaults/baileys-version.json (hunk 1 above)
Comment (rzkgnz) [reply src/Defaults/baileys-version.json]: 😅 I forgot I followed the wa web version

Comment (edgardmessias) [reply src/Defaults/baileys-version.json]: After this merge, I will update the "proto update" workflow to update this file too

**Pull request merged**